### DISCUSSION TITLE
PrusaSlicer Discussions!

### DISCUSSION THREAD

**Discussion by leandrolima-nyc** [leandrolima-nyc/SnapmakerJ1#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and suggestions.
  * Engage with other community members.
 
Remember that this is a collaborative community that we are building together. It is essential to be considerate, respectful, and open-minded toward one another 💪

To get started, kindly introduce yourself in the comments section, including the operating system you are using and the version of PrusaSlicer you have installed. Additionally, please share your experience with the profile so far.

**Comment by Mircus86** [comment]
hi Leandro, i am using your profile for j1 in prusa alpha 6 ( used in alpha 5 too) everything is wonderful.
i only have a problem in copy mode.
it preheats everything, but the copy doesn't start

do you have any idea of why happens?
